plugins/swiot/src/runtime/ad74413r/buffermenu.hpp

```diff
@@ -0,0 +1,254 @@
+#ifndef SWIOTADVMENU_H
+#define SWIOTADVMENU_H
```

Comment:
For multiple files that were renamed in the dev process, this macro does not match the class and the file name
I guess it would be easier if we would have them match.